### ISSUE TITLE
Value of type TTTAttributedLabel has no member named setText

### ISSUE THREAD

**Issue by happiehappie** [TTTAttributedLabel/TTTAttributedLabel#672]
I updated TTTAttributedLabel to 2.0.0 and now I get the above error. I checked the documentation and Example project and I can't figure out what's wrong, am I missing something?

**Comment by happiehappie** [comment]
I don't think it matters but just to provide more info, I'm using Swift, XCode 7.3 instead of O-C

**Comment by AvdLee** [comment]
+1 on this one. It works on my local machine, stops working on my buildserver
